src/lib/datepicker/calendar.html

```diff
@@ -24,18 +24,17 @@
   <mat-month-view
       *ngSwitchCase="'month'"
       [(activeDate)]="_activeDate"
-      [selected]="selected"
+      [selectionModel]="selectionModel"
```

Comment:
I think we can still call the input `selected` and just pass the selection model instead now